File: JanperCab/3-Application/Dtos/DuraformMiscComponent/DuraformMiscComponentPriceDto.cs
namespace _3_Application.Dtos.DuraformMiscComponent
{
    public class DuraformMiscComponentPriceDto
    {
        public int Id { get; set; }

        public decimal UnitPrice { get; set; }

        public decimal SubTotal { get; set; }

        public decimal TotalDiscount { get; set; }

        public decimal TotalPrice { get; set; }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-misc-component/DuraformMiscComponentPriceDto.ts
import { DuraformMiscComponentDto } from 'src/app/_models/duraform-misc-component/DuraformMiscComponentDto';
export class DuraformMiscComponentPriceDto {
  id: number;

  unitPrice: number;
  subTotal: number;
  totalDiscount: number;
  totalPrice: number;

  constructor(duraformMiscComponent: DuraformMiscComponentDto) {
    this.id = duraformMiscComponent.id;
    this.unitPrice = duraformMiscComponent.unitPrice;
    this.subTotal = duraformMiscComponent.subTotal;
    this.totalDiscount = duraformMiscComponent.totalDiscount;
    this.totalPrice = duraformMiscComponent.totalPrice;
  }
}
